tests/unittests/basicIRTest.cpp

```diff
@@ -131,7 +131,7 @@ TEST(IR, allInstrs) {
 
     builder.createCopyInst("", I1, I0);
     builder.createConvolutionInst("", I3, I1, F0, B0, 7, 2, {3, 3, 3, 3}, 1);
-    builder.createPoolMaxInst("", I4, I0, 7, 2, 3);
+    builder.createPoolMaxInst("", I4, I0, 7, 2, {3, 3, 3, 3});
```

Comment:
Don't we still have a version that takes scalar pads?

Reply:
Yes, we do have scalar pads, but that is for Node level.  At instruction level,  I don't think it is necessary to keep the scalar pad :)